source/nodejs/adaptivecards/src/serialization.ts

```diff
@@ -107,6 +107,7 @@ export class Versions {
     static readonly v1_1 = new Version(1, 1);
     static readonly v1_2 = new Version(1, 2);
     static readonly v1_3 = new Version(1, 3);
+    static readonly v1_5 = new Version(1, 5);
```

Comment:
You can add labels to Version instances, as the third constructor parameter. Could you add "Preview" as the label?